### DISCUSSION TITLE
FormField make default value avaliable in __get function.

### DISCUSSION THREAD

**Discussion by peter1szalatnay** [joomla/joomla-cms#47037] [2. Cold Features]
### Is your feature request related to a problem? Please describe.

It would be good to be able to check if the current selected form field value is the default or not.

Example:
```
$active = $field->value != $field->default ? true : false;
```
Right now default always return null as it is not in the allowed __get list.

### Describe the solution you'd like

Add 'default' to the switch case for FormField::__get function.

**Comment by joomdonation** [comment]
You can already get default value of the field using `getAttribute` method. For example `$default = $field->getAttribute('default');`
